### ISSUE TITLE
Fails on Install

### ISSUE THREAD

**Issue by roosnic1** [truenorth/ember-material-lite#8]
Hi I'm working with ember-cli 1.13.1 and I'm trying to install ember-material-lite into a fresh new ember project. I get the following error:

```sh
ember install ember-material-lite
version: 1.13.1
Installed packages for tooling via npm.
installing ember-material-lite
The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.
```

I googled the error and read that it is a silent ember-cli error but my ember-material-lite still doesn't work. I saw in issue #6  that someone has a similar problem. 

I'm working with OSX 10.10.4. Can you help me?

**Comment by truenorth** [comment]
I had to make a fix to ember-cli in order to make this blueprint work, but it hasn't yet been released. 

In the mean time, you can add the following to bower.json

```js
'material-design-lite', '1.0.0',
'material-design-lite-src', 'https://github.com/google/material-design-lite.git#37286ae139b0ae336a8a7e9ed3b18ace1abff7ed'
```

add this to your package.json
```js
'ember-cli-sass', '^3.3.0',
```

And add this to your app/index.html

```html
<link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
```

**Comment by roosnic1** [comment]
Hi thanks a lot for your answer. I already tried that because of the last comment in issue #6  but I still get the following error when I run ```ember serve```:

```sh
{
  "status": 1,
  "file": "/Users/nicolas/tmp/playwrite/tmp/tree_merger-tmp_dest_dir-MyoFKbS0.tmp/app/styles/app.scss",
  "line": 6,
  "column": 9,
  "message": "file to import not found or unreadable: material-design-lite\nCurrent dir: /Users/nicolas/tmp/playwrite/tmp/tree_merger-tmp_dest_dir-MyoFKbS0.tmp/app/styles/"
} [string exception]
```

My app.scss looks like the one described in the project:

```scss
/*@import "_color-definitions";

$color-primary: $palette-teal-500;
$color-accent: $palette-pink-A200;*/
@import "material-design-lite";
```

Do I have to build something first?

**Comment by rtablada** [comment]
@truenorth (yಠ,ಠ)y  paste valid JSON to add to suggested `Bower.json`

**Comment by roosnic1** [comment]
@truenorth Do you have any news? Or is the ember-cli update coming soon? 

Thanks for your help.

**Comment by truenorth** [comment]
The pertinent PR is already merged into ember-cli
https://github.com/ember-cli/ember-cli/pull/4430

We'll release a new ember-cli version soon 
![screen shot 2015-07-23 at 10 54 02 am](https://cloud.githubusercontent.com/assets/558005/8857868/2c9f3770-3129-11e5-9f80-1894887312b1.png)

**Comment by roosnic1** [comment]
Thanks for the answer. This might be a stupid question but can I use the master branch version of ember-cli already today or should I wait for the release?

**Comment by truenorth** [comment]
you could use master of ember-cli if you wanted to. It's just a very complex node.js command line tool

**Comment by alexjp** [comment]
@roosnic1 i think you are missing 
```
  sassOptions: {
      includePaths: ['bower_components/material-design-lite-src/src']
    },
```
in config/environment.js

**Comment by truenorth** [comment]
@alexjp it's a little worse than that. The current version of ember-cli (1.13.1) only supports a subset of valid bower package name/version descriptions. It barfs at this one

```
"material-design-lite-src": "https://github.com/google/material-design-lite.git#37286ae139b0ae336a8a7e9ed3b18ace1abff7ed"
```
Which is the only means of installing the sass via a blueprint (the version published to npm and bower both include only css).

**Comment by alexjp** [comment]
@truenorth got it, and i suppose that the new ember-cli will bring that ( with your patch ).

By the way, in my development, i upgraded to mdl 1.0.1. ( all seems to worked out fine )

**Comment by roosnic1** [comment]
@alexjp and @truenorth so my only solution is to wait till the next ember-cli release? Any idea if it's gonna be days or weeks?

Thanks

**Comment by alexjp** [comment]
@roosnic1 i manager to mais it work. But you havê to do it by hand. The tipos here should be enough

**Comment by truenorth** [comment]
Yay! Looks like MDL v1.0.1 includes the sass source files and individual JS modules. Looks like we may get a reasonable blueprint without waiting for ember-cli now

**Comment by henriquecf** [comment]
I am sorry but this has not worked for me here. Still getting: The `ember generate` command requires an entity name to be specified. For more details, use `ember help`.

**Comment by truenorth** [comment]
The way that the "default" blueprint for an addon currently works -- it is either invoked as part of `ember install ember-material-lite`, or you can invoke it yourself by running `ember generate ember-material-lite my-app` (or whatever the name of your app is)

**Comment by henriquecf** [comment]
Thanks @truenorth that worked perfectly!!

**Comment by roosnic1** [comment]
@mike-north I finally found the time to retry it. It worked with ```ember install ember-material-lite``` and ```ember generate ember-material-lite my-app```. But as soon as I used a component that needed "ember-composability/mixins/parent-component-support" the loading of the page failed with an error in the javascript console that it could not load the module. I then installed ember-composability manually and everything worked. Just FYI.

Thanks again for the help.
